spec/services/dialog_local_service_spec.rb

```diff
@@ -21,6 +21,19 @@
         :cancel_endpoint        => "/catalog/explorer"
       )
     end
+
+    it "uses the base class for service_template derived classes" do
+      target_ansible = instance_double("ServiceTemplateAnsiblePlaybook", :class => ServiceTemplateAnsiblePlaybook, :id => 987, :service_template_catalog_id => 798)
+      expect(service.determine_dialog_locals_for_svc_catalog_provision(resource_action, target_ansible, finish_submit_endpoint)).to eq(:resource_action_id     => 456,
+                                                                                                                                       :target_id              => 987,
+                                                                                                                                       :target_type            => 'service_template_ansible_playbook',
```

Comment:
Shouldn't the target type end up being just `service_template` and not `service_template_ansible_playbook`? Otherwise this doesn't fix the issue, because the API doesn't know what a `service_template_ansible_playbook` collection is.

Also it seems weird to me that rubocop didn't flag the line length issue, but you could put the hash on a new line so it looks something like:
```
expect(method_call).to eq(
  :resource_action_id => "123",
  :target_id          => "321",
  # etc
)